templates/competitorResults.html

```diff
@@ -1,17 +1,31 @@
 <template name="participantResults">
   <div class="container">
-
-    <ul>
-      {{#each results}}
-        <li>
-          {{! TODO - perhaps combine this and the code in roundResults into a template? }}
-          {{#each solves}}
-            {{clockFormat this}}
-          {{/each}}
-          best: {{clockFormat best}}
-        </li> 
-      {{/each}}
-    </ul>
+    <h1>{{competitorName}}</h1>
+    <h3><a href={{wcaProfileUrl}}>WCA Profile</a></h3>
```

Comment:
Just left it as is (for now), because we talked about not dealing with backward compatibility in the futuree